lib/cli/commands/addcurrency.ts

```diff
@@ -0,0 +1,38 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { AddCurrencyRequest } from '../../proto/xudrpc_pb';
+import { NetworkClients } from '../../types/enums';
+
+export const command = 'addcurrency <currency> <network_client> [token_address] [subunits]';
+
+export const describe = 'add a supported currency';
+
+export const builder = {
+  currency: {
+    description: 'The ticker symbol for the currency',
+    type: 'string',
+  },
+  network_client: {
+    description: 'The payment channel network client',
+    type: 'string',
+    choices: ['LND', 'RAIDEN'],
+  },
+  token_address: {
+    description: 'The contract address for layered tokens such as ERC20',
+    type: 'string',
+  },
+  subunits: {
```

Comment:
I am fine with `subunits` but did you consider using the decimals of the currency instead?

Reply:
What do you mean? Like the base 10 exponent for the number of subunits in a unit? Such that bitcoin's would be 8? 10^8 satoshis in a bitcoin.

Reply:
Yeah exactly. Especially because all of those ERC20 contracts are using that format and copying `100000000000000000` is harder than just `18`.

Reply:
Yeah I agree that's better, I'll change it.